Dockerfile

```diff
@@ -4,38 +4,40 @@ ENV USER l4d2
 ENV HOME /home/$USER
 ENV SERVER $HOME/hlserver
 
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
 RUN apt-get -y update \
-    && apt-get -y upgrade \
-    && apt-get -y install lib32gcc1 curl net-tools lib32stdc++6 locales unzip lib32z1 screen \
+    && apt-get --no-install-recommends -y install lib32gcc1 curl net-tools lib32stdc++6 locales unzip lib32z1 screen \
     && locale-gen en_US.UTF-8 \
     && update-locale LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
-    && dpkg-reconfigure --frontend=noninteractive locales \
-    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-    && useradd $USER \
+    && dpkg-reconfigure --frontend noninteractive locales \
+    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN useradd $USER \
     && mkdir $HOME \
     && chown $USER:$USER $HOME \
-    && mkdir $SERVER
+    && mkdir $SERVER \
+    && chown --recursive $USER:$USER $SERVER
```

Comment:
Should we just move the mkdir server down to under the `USER $USER` section?

Reply:
That would work. I think `useradd` has options that should obviate the `chown`ing of home directories also. My work day just started but I'll look into that more later tonight.

Reply:
8e4eab32f8cc99d3e9f1c9c2c1415cfbcbd47d25 

When I build this, I'm getting errors about invalid SSL certs or params or something. But I don't think (?) it's about the changes I've made here. Let me know what you think.